Move dev gems to dev dependency group

henrich-m opened the pull request:
Why? Coveralls and webmock are used only for testing.

Also:
* Some code :lipstick:
* Released a new gem version
* Uploaded the new version to jfrog.io

henrich-m committed:
Move coveralls to dependency map

Why? Coveralls is used only in testing

Also apply some code :lipstick:

henrich-m committed:
Bump version

andrehjr commented:
LGMT

nsoufr commented on pipedrive-ruby.gemspec:
```diff
@@ -2,112 +2,112 @@
+      s.add_runtime_dependency('json', ['>= 1.7.7'])
+      s.add_runtime_dependency('multi_xml', ['>= 0.5.2'])
+      s.add_runtime_dependency('webmock', ['>= 0'])
+      s.add_development_dependency('bundler', ['>= 1.0.0'])
```
Se esta no Gemfile nem precisa estar no gemspec :)

henrich-m replied:
:eyes:

henrich-m committed:
Fix gems source

henrich-m committed:
Webmock is required only in testing too

* Also simplify the gemspec as instruced by @nando in code review

henrich-m committed:
Enforce jfrog and block public pushes

henrich-m commented on Gemfile:
```diff
@@ -1,4 +1,4 @@
-source "http://rubygems.org"
+source 'http://rubygems.org'
```
:heavy_check_mark:

nsoufr replied:
Não acho que seja uma boa pessoal.

Jfrog a gente deve usar no projeto e não na criação de gems. 

A gem pode sim buildar e dar push para o jfrog, mas não vejo pq O Gemfile (que é ignorado na app host) ter o jfrog como source.

henrich-m commented:
@nandosousafr pode dar uma segunda olhada ?

henrich-m committed:
Bump gem version

henrich-m committed:
fixup! Fix gems source

henrich-m committed:
Use travis ci for testing gem

henrich-m committed:
Revert gems source to rubygems

henrich-m committed:
Explicit test-unit dependency

nsoufr reviewed (approved):
LGTM

nsoufr commented on Gemfile:
```diff
@@ -1,20 +1,21 @@
-source "http://rubygems.org"
+source 'https://rubygems.org'
```
que rapido, mal pude ver seus movimentos

henrich-m replied:
no JFrog ele vai pedir user/pass para rodar no CI, como a gem é publica eu acho que não faz sentido, o que acha ?

nsoufr replied:
perfeito. não precisa na gem :)

henrich-m committed:
Fix requests stubs urls

Why ? As described in docs, all comunication with pipedrive should
be in https only.

https://github.com/ResultadosDigitais/pipedrive-ruby/blob/master/lib/pipedrive/base.rb#L19

henrich-m committed:
Bump gem version
